src/main/java/org/pac4j/vertx/verticle/DemoServerVerticle.java

```diff
@@ -121,7 +121,13 @@ public void start() {
         // Requires authentication endpoint without specific authenticator attached
         addProtectedEndpointWithoutAuthorizer("/protected/index.html", "", router);
 
-        router.get("/index.html").handler(DemoHandlers.indexHandler(config));
+        // Direct basic auth authentication (web services)
+        addProtectedEndpointWithoutAuthorizer("/dba/index.html", "DirectBasicAuthClient", router);
```

Comment:
The first endpoint is protected by two possible clients (/dba): token or basic auth, the second one only by one client (/rest-jwt): token.
To get a token, you must login using an indirect client like Facebook and then click on "Generate jwt" (/jwt). The value is your token. Logout after you have saved the token somewhere to start a fresh test.
Then, you can call one of the two protected urls (both supports token) with no query string at first, you should be unauthorized. Then, you can add a `?token=` parameter (no token value) and see that it generates an error (indeed, a blank token fails to be properly decoded). Finally, you can use `?token=the_generated_token_you_get_previously` to see that you get the same profile as the one you had when you generated the token.
You can try to force the client used with a false value (`?client_name=FakeClient`) and see that you get an error or with a true value for the endpoint which supports two clients: `/dba/index.html?token=the_generated_token_you_get_previously` works, but `?client_name=DirectBasicAuthClient&token=the_generated_token_you_get_previously` should return an unauthorized error (indeed, if you want to use basic auth, a token is useless). `?client_name=ParameterClient&token=the_generated_token_you_get_previously` works as well.
To test basic auth, you need some extension / plugin in your browser to be able to push the `Authorization` header with the appropriate value: `Basic amxlbGV1OmpsZWxldQ==` (as stated on the index page) as a POST request. Use `client_name=ParameterClient` to see that even with the right header, it doesn't work (indeed, the `ParameterClient` expects a token and not basic auth).